roles/openshift_etcd_facts/tasks/set_etcd_ca_host.yml

```diff
@@ -38,6 +39,16 @@
   - __etcd_ca_hosts | length > 0
   - etcd_ca_host is not defined
 
+# Fail early on redeployment of certificates if /etc/etcd/ca directory is missing
+- fail:
+    msg: >
+      CA indicator files and directories does not exist on CA hosts.
+      Apply 'etcd_ca' action from `etcd` role.
```

Comment:
Although it makes most sense in what regards openshift-ansible internals, "apply 'etcd_ca' action from `etcd` role" is something a customer will not understand. Not sure if a more generic message like "redeploy etcd CA" would make more sense.

An alternative would be to keep the message as is and document its meaning separately.